scripts/_common.sh

```diff
@@ -70,10 +70,14 @@ EOT
         pgloader $tmpdir/commands.load
 
         # Rebuild INDEX
-        ynh_psql_execute_as_root --sql='CREATE INDEX idx_fileinfo_content_txt ON public.fileinfo USING gin (to_tsvector('\''english'\''::regconfig, content))' --database=mattermost
-        ynh_psql_execute_as_root --sql='CREATE INDEX idx_posts_message_txt ON public.posts USING gin (to_tsvector('\''english'\''::regconfig, (message)::text));' --database=mattermost
+        ynh_psql_execute_as_root --sql='CREATE INDEX idx_fileinfo_content_txt ON public.fileinfo USING gin (to_tsvector('\''english'\''::regconfig, content))' --database=$db_name
+        ynh_psql_execute_as_root --sql='CREATE INDEX idx_posts_message_txt ON public.posts USING gin (to_tsvector('\''english'\''::regconfig, (message)::text));' --database=$db_name
 
 
+        if ynh_compare_current_package_version --comparison eq --version 7.3.0~ynh1
+        then
+            ynh_psql_execute_as_root --sql="DELETE FROM public.db_migrations;" --database=$db_name
```

Comment:
Yikes! Maybe that's the way to unstuck the data migration though. Why was this necessary?

Reply:
I just found [this](https://github.com/mattermost/mattermost-server/issues/18351#issuecomment-1325648058) and tryied it. There is probably a less brutal way to do it though

Reply:
https://forum.mattermost.com/t/migrating-from-mariadb-to-postgresql-db/14194/6

Maybe we can try to do this instead
```
DELETE FROM db_migrations WHERE version=92;
```

Reply:
Ok, i see where it comes from. I suggest:

- Only removing a single migration, like your example,
- Adding a link to the relevant forum thread in comment 

and then I think we can happily merge this!